oak_restricted_kernel/src/lib.rs

```diff
@@ -95,13 +98,22 @@ pub fn start_kernel(info: &BootParams) -> Box<dyn Channel> {
     let mut mapper = mm::init_paging(&mut frame_allocator, program_headers).unwrap();
 
     // Allocate a section for guest-host communication (without the `ENCRYPTED` bit set)
-    let guest_host_frame: PhysFrame<Size2MiB> = frame_allocator.allocate_frame().unwrap();
-    let guest_host_page = mapper.translate_physical_frame(guest_host_frame).unwrap();
-    let _guest_host_heap = unsafe {
-        memory::init_guest_host_heap(
-            Page::range(guest_host_page, guest_host_page + 1),
-            &mut mapper,
-        )
+    let guest_host_frames = frame_allocator.allocate_contiguous(2).unwrap();
```

Comment:
Maybe it's worth commenting what "2" we are allocating (and why 2)?

Reply:
Good point, done.